docs/source/concepts/overview.mdx

```diff
@@ -0,0 +1,7 @@
+# Concepts
+
+In this section, you will find high-level explanations for building a better understanding
+of important topics such as the philosophy behind `huggingface_hub`, the git-based vs http-based
+paradigm or the cache system internals.
+
+This section is currently empty but will get populated soon.
```

Comment:
Are you sure about adding a new empty page to be populated later? I would add it when there is an actual need to reduce the noise

Reply:
I agree it may be better to leave the "coming soon" docs out for now until they're ready. In addition to noise, it also creates expectations from users, and when the new doc isn't published soon some users may be disappointed.

Reply:
I get your point. I removed them in https://github.com/huggingface/huggingface_hub/pull/1233/commits/dd954923763037b33bc1387f52e8367ba50d5fab.